app/views/idv/in_person/state_id.html.erb

```diff
@@ -61,7 +61,7 @@
         ) %>
   </div>
 
-  <div class="tablet:grid-col-7 margin-bottom-5">
+  <div class="tablet:grid-col-8 margin-bottom-5">
```

Comment:
just checking - what ds is this from?

Reply:
These are U.S. Web Design System utility classes: https://designsystem.digital.gov/utilities/layout-grid/